qemu/tests/in_place_upgrade_legacy.py

```diff
@@ -0,0 +1,126 @@
+import re
+import logging
+import aexpect
+
+
+from virttest import error_context
+from qemu.tests.in_place_upgrade import IpuTest
+
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+class IpuLegacyTest(IpuTest):
+    """
+    Provide basic functions for in place upgrade test cases
+
+    """
+    def __init__(self, test, params, env):
+
+        super(IpuTest, self).__init__()
+        self.session = None
+        self.test = test
+        self.params = params
+        self.env = env
+
+    def pre_upgrade_whitelist(self, test):
+        """
+        Fix known issues after executing pre-upgrade
+
+        """
+        try:
+            # Possible problems with remote login using root account
+            self.session.cmd(self.params.get("fix_permit"))
+            # Answer file missing will be fixed
+            fix_answer = self.params.get("fix_answer_file")
+            self.session.cmd(fix_answer, timeout=1200)
+            fix_answer_sec = self.params.get("fix_answer_section")
+            self.session.cmd(fix_answer_sec, timeout=1200)
+            erase_old_kernel = self.params.get("clean_up_old_kernel")
+            s, output = self.session.cmd_status_output(erase_old_kernel,
+                                                       timeout=1200)
+            error_info = self.params.get("error_info")
+            if re.search(error_info, output):
+                pass
+        except Exception as info:
```

Comment:
Print error messages are helpful for debugging.

Reply:
will update, thanks

Reply:
will update, thanks.